### PR TITLE
Changed the year of creation

### PR DESCRIPTION
**Pull request by atanaskova**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -112,7 +112,7 @@ <h2>HAVE A PROJECT?</h2>
         </form>
     </div>
     <footer>
-        Copyright &copy; 2019 Mihajlo Kragujevski
```
Comment (KMihajlo) [inline index.html:115]: This should be 2019, as that is the year this project has been started.

File: index.html [hunk 2]
```diff
@@ -112,7 +112,7 @@ <h2>HAVE A PROJECT?</h2>
...
+        Copyright &copy; 2021 Mihajlo Kragujevski
```
Comment (atanaskova) [inline index.html:115]: 2021 or 2022?

File: index.html (hunk 1 above)
Comment (atanaskova) [reply index.html:115]: Okay no problem

Comment (atanaskova) [reply index.html:115]: Gonna change that as soon as possible

**Pull request closed without merging**